packages/design-system/src/components/StepList/Step.jsx

```diff
@@ -75,7 +76,12 @@ export const Step = ({ step, ...props }) => {
         )}
       </div>
       <div className="ds-c-step__actions" {...actionsLabel}>
-        {step.completed && <div className="ds-c-step__completed-text">{props.completedText}</div>}
+        {step.completed && (
+          <div className="ds-c-step__completed-text">
+            <CheckIcon className="ds-c-icon-color--success" />
```

Comment:
There's a difference in placement:

![steplist](https://user-images.githubusercontent.com/7595652/147165986-a20a4826-68f3-42d8-b772-ead427d334da.gif)